tests/init_test.go

```diff
@@ -93,7 +93,7 @@ type testMatcher struct {
 	failpat        []testFailure
 	skiploadpat    []*regexp.Regexp
 	slowpat        []*regexp.Regexp
-	runonlylistpat *regexp.Regexp
+	runonlylistpat []*regexp.Regexp
```

Comment:
make `runonlylistpat` is a list of patterns, so we can add more than one pattern